crates/dkg-cli/src/actions.rs

```diff
@@ -333,11 +344,13 @@ async fn wait_for_phase<P: JsonRpcClient, S: Signer>(
     Ok(())
 }
 
-fn parse_bundle<D: serde::de::DeserializeOwned>(bundle: &[Vec<u8>]) -> Result<Vec<D>> {
+fn parse_bundle<D: serde::de::DeserializeOwned>(
+    bundle: &[ethers::prelude::Bytes],
+) -> Result<Vec<D>> {
     bundle
         .iter()
-        .filter(|item| !item.is_empty()) // filter out empty items
-        .map(|item| Ok(bincode::deserialize::<D>(&item)?))
+        .filter(|item| !item.to_vec().is_empty()) // filter out empty items
```

Comment:
I cannot get the build to work, but maybe you can replace `!item.to_vec().is_empty()` with `!item.is_empty()`, which would reduce allocations.